### DISCUSSION TITLE
[Idea] 话题模板

### DISCUSSION THREAD

**Discussion by arvinxx** [lobehub/lobehub#869] [Ideas]
类似相关需求

- https://github.com/lobehub/lobe-chat/issues/588
- #336
- https://github.com/lobehub/lobe-chat/issues/1370

我也知道只要给出有效的上下文，可以优化和引导大模型的输出。而且我之前也做了一个组件可以实现上述功能 https://ui.lobehub.com/components/editable-message-list

但这个功能一直没有实装进 lobe-chat中。原因是我觉得这个方案过于技术视角了。虽然管用的，但是体验上非常不好用。

经过最近的一段时间的深度使用lobechat 测试工程师，对于这个场景有了一些新的启发。记录下来后续实现。

即我们可以将过去一段时间的上下文消息创建为一个话题模板。这样一来用户可以再已有的模板基础上进行会话。在话题模板下会话，会自动创建出新的话题。这样对于某些重复场景来说就会比较方便。

同时，我们可以把话题模板作为这个助手的特色能力分享到市场。这样一来，助手就不仅仅是一个单纯通过prompts驱动的助理。通过话题模板的形式可以更加深入地定制助手的特色功能。

**Comment by bmwa0813** [comment]
赞。一个助手下可以拥有多个话题，说起来类似glow的梦境

**Comment by Lostsite** [comment]
- #2170 

Also suggest to add this suggestion to "Topic Template" function
